templates/oio-event-handlers.conf.erb

```diff
@@ -49,6 +49,14 @@ use = egg:oio#content_cleaner
 use = egg:oio#notify
 tube = oio-rebuild
 queue_url = <%= @_rebuild_queue_url %>
+
+[handler:storage.content.perfectible]
+pipeline = content_improve
```

Comment:
A default handler for `storage.content.perfectible` events should be defined in any case, or the logs will be spammed with `WARNING no handler found for [...]` messages.

A good default handler would be like:
```
[handler:storage.content.perfectible]
pipeline =
```
No filter in the pipeline means "don't do anything with the event".